flink-scala/pom.xml

```diff
@@ -236,4 +230,23 @@ under the License.
 		</plugins>
 	</build>
 
+	<profiles>
+		<profile>
+			<id>scala-2.10</id>
+			<activation>
+				<property>
+					<!-- this is the default scala profile -->
+					<name>!scala-2.11.profile</name>
```

Comment:
Another argument for activating profiles by their profile name is IDE support: In IntelliJ its very easy to activate them in the profiles section of the Maven view.
For property activated profiles, you have to go into the maven settings.